test/elementhandle-convenience.spec.ts

```diff
@@ -176,6 +176,8 @@ it('isEnabled and isDisabled should work', async ({ page }) => {
     <button disabled>button1</button>
     <button>button2</button>
     <div>div</div>
+    <div role="button" aria-disabled="true">role=button1</div>
+    <div role="button" aria-disabled="false">role=button2</div>
```

Comment:
Let's add a testcase with `aira-disabled="foo"`, and change false case to `aria-disabled="faLSE"`